app/controllers/invitation_controller.rb

```diff
@@ -0,0 +1,20 @@
+class InvitationController < ApplicationController
+
+  def new
+
+  end
+
+  def create
+    github_service = GithubService.new(current_user)
+    user = github_service.get_user(params[:handle])
+    github_user = GithubUser.new(user)
```

Comment:
Would it still be clear enough if these 3 lines were refactored into 1: github_user = GithubUser.new(GithubService.new(current_user).get_user(params[:handle])) ?

Reply:
Maybe something in the middle to preserve more readability?

Reply:
Just changed this fyi. I think it's a good middle ground, let me know what you think

Reply:
Looks good